Content.Client/Storage/Visualizers/StorageVisualizerSystem.cs

```diff
@@ -0,0 +1,93 @@
+using Content.Shared.Storage;
+using Robust.Client.GameObjects;
+
+namespace Content.Client.Storage.Visualizers;
+
+public sealed class StorageVisualizerSystem : VisualizerSystem<StorageVisualizerComponent>
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<StorageVisualizerComponent, ComponentInit>(OnInit);
+    }
+
+    /// <summary>
+    /// Sets the base sprite to this layer. Exists to make the inheritance tree less boilerplate-y.
+    /// </summary>
+    private void OnInit(EntityUid uid, StorageVisualizerComponent comp, ComponentInit args)
+    {
+        if(!TryComp<SpriteComponent>(uid, out var sprite))
+            return;
+
+        if (comp.StateBase != null)
+        {
+            sprite.LayerSetState(0, comp.StateBase);
+        }
+
+        if (comp.StateBaseAlt == null)
+        {
+            comp.StateBaseAlt = comp.StateBase;
+        }
```

Comment:
cursed way to handle nullability, its not even necessary considering there is a null check before any StateBaseAlt reference